ppml/README.md

```diff
@@ -163,7 +159,51 @@ https://user-images.githubusercontent.com/61072813/184758702-4b9809f9-50ac-425e-
 To secure your Big Data & AI applications in BigDL PPML manner, you should prepare your environment first, including K8s cluster setup, K8s-SGX plugin setup, key/password preparation, key management service (KMS) and attestation service (AS) setup, BigDL PPML client container preparation. **Please follow the detailed steps in** [Prepare Environment](./docs/prepare_environment.md). 
 
 
-#### Step 1. Encrypt and Upload Data
+#### Step 1. Build your PPML image for production environment
+To build a secure PPML image which can be used in production environment, BigDL prepared a public base image that does not contain any secrets. You can customize your own image on top of this base image.
+
+1. Build BigDL Base Image
```

Comment:
change to Prepare BigDL Base Image
users can pull the base image or build it by themselves